src/js/serial_backend.js

```diff
@@ -43,11 +44,8 @@ function initializeSerialBackend() {
     $('div.connect_controls a.connect').click(function () {
         if (GUI.connect_lock != true) { // GUI control overrides the user control
 
-            const thisElement = $(this);
-            const clicks = thisElement.data('clicks');
```

Comment:
Another fix for this can be:
```javascript
const clicks = thisElement.data('clicks') || false;
```
but I think is better to use variables and not attach variables to html elements.